app/auto-eligibility/db/customer.db.table.js

```diff
@@ -48,12 +48,12 @@ const updateWaitingUpdatedAt = async (sbi, crn) => {
 }
 
 const updateAccessGranted = async (upperLimit) => {
```

Comment:
while we're changing this method we could add some more logging to it, similar to:

<img width="835" alt="image" src="https://user-images.githubusercontent.com/118533775/212631973-55b1b7d0-839c-49a9-849d-4e087f5c9300.png">